app/models/user.rb

```diff
@@ -24,14 +22,17 @@ class User < ApplicationRecord
     HOMESCHOOL_ROLE    = :homeschool
   ].freeze
 
+  # TODO: Thinking we can migrate to having faculty status == state - instead of having both
   VALID_FACULTY_STATUSES = [
     NO_FACULTY_INFO = 'no_faculty_info',
     PENDING_FACULTY = 'pending_faculty',
     CONFIRMED_FACULTY = 'confirmed_faculty',
     REJECTED_FACULTY = 'rejected_faculty',
     PENDING_SHEERID = 'pending_sheerid',
     REJECTED_BY_SHEERID = 'rejected_by_sheerid',
-    INCOMPLETE_SIGNUP = 'incomplete_signup'
```

Comment:
Since this is used to create an enum, users that had incomplete_signup status before will automatically be updated to needs_email_verified which may not be what we want.

Reply:
We haven't released the incomplete_signup code yet 😅 - so hopefully not an issue?